Interface-Signoff-Marvo.md

```diff
@@ -0,0 +1,48 @@
+# Interface Signoff
+
+## Subsystem Function
+The interface conditions any input analog voltage, analog current and digital signal then transmits it to the microcontroller. 
+## Subsystem Constraints
+1) Must be able to operate with mV voltage output ranges
```

Comment:
1-4. Clarify what is input to this subsystem and what is output from this subsystem. 1. Need to specify range, "mV" by itself is too vague. 5. What purpose does this serve? X. It appears that output voltage range is not specified so that needs to be added to the constraints.

Reply:
OK except constraint #4 needs a voltage tolerance and a maximum current.